homeassistant/components/songpal/media_player.py

```diff
@@ -120,10 +125,11 @@ async def async_service_handler(service):
 class SongpalDevice(MediaPlayerDevice):
     """Class representing a Songpal device."""
 
-    def __init__(self, name, endpoint, poll=False):
+    def __init__(self, name, endpoint, model=None, poll=False):
         """Init."""
         self._name = name
         self._endpoint = endpoint
+        self._model = model
```

Comment:
Model could be queried from the device during the initial `update()`, if that makes sense?